.storybook/main.js

```diff
@@ -0,0 +1,21 @@
+module.exports = {
+  stories: ['../**/*.stories.js'],
+  addons: [
+    {
+      name: '@storybook/addon-essentials',
+      options: {
+        // we are not ready to use them yet.
+        controls: false,
+        docs: false
+      }
+    },
+    {
+      name: '@storybook/addon-storysource',
+      options: {
+        parser: 'typescript'
+      }
+    },
+    '@storybook/addon-a11y',
+    'storybook-readme'
```

Comment:
`storybook-readme` should move to the top of addons array in order to show README tab first.
#1041